src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -1,21 +1,20 @@
 package cromwell.engine.db
 
-import cromwell.engine.WorkflowOutputs
-import wdl4s._
 import cromwell.engine.ExecutionStatus.ExecutionStatus
-import cromwell.engine._
+import cromwell.engine.{WorkflowOutputs, _}
 import cromwell.engine.backend.{Backend, JobKey}
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.{BackendCallKey, ExecutionStoreKey, OutputKey}
 import cromwell.webservice.{CallCachingParameters, WorkflowQueryParameters, WorkflowQueryResponse}
+import wdl4s._
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 object DataAccess {
   val globalDataAccess: DataAccess = new slick.SlickDataAccess()
 }
 
-trait DataAccess {
+trait DataAccess extends AutoCloseable {
```

Comment:
Man I'm so confused :)  I've never seen an `AutoCloseable` before (I'm guessing it's [this](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html)).  I read the API but I guess I'm still not clear on why it was added!